src/main/java/com/github/maven_nar/cpptasks/msvc/Assembler64bitCompiler.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * 
+ * Copyright 2002-2004 The Ant-Contrib project
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.github.maven_nar.cpptasks.msvc;
+
+import com.github.maven_nar.cpptasks.CUtil;
+import com.github.maven_nar.cpptasks.OptimizationEnum;
+import com.github.maven_nar.cpptasks.compiler.CommandLineCompiler;
+import com.github.maven_nar.cpptasks.compiler.LinkType;
+import com.github.maven_nar.cpptasks.compiler.Linker;
+import com.github.maven_nar.cpptasks.compiler.Processor;
+import com.github.maven_nar.cpptasks.parser.CParser;
+import com.github.maven_nar.cpptasks.parser.Parser;
+import org.apache.tools.ant.types.Environment;
+
+import java.io.File;
+import java.util.Vector;
+
+/**
+ * Adapter for the Microsoft (R) Macro Assembler (x64)
+ * 
+ * @author Curt Arnold
+ */
+public final class Assembler64bitCompiler extends CommandLineCompiler {
+    private static final Assembler64bitCompiler instance = new Assembler64bitCompiler(
+            false, null);
+    public static Assembler64bitCompiler getInstance() {
+        return instance;
+    }
+    private String identifier;
+    private Assembler64bitCompiler(boolean newEnvironment, Environment env) {
+        super("ml64", null, new String[]{".asm"}, null, ".obj", false, null, newEnvironment, env);
```

Comment:
Rather than null list of headers that requires change to Abstract class pass `new String[] {}`  ie. 
`super("ml64", null, new String[]{".asm"}, new String[] {}, ".obj", false, null, newEnvironment, env);`